web_src/js/features/common-organization.ts

```diff
@@ -3,17 +3,18 @@ import {queryElems, toggleElem} from '../utils/dom.ts';
 import {initAvatarUploaderWithCropper} from './comp/Cropper.ts';
 
 export function initCommonOrganization() {
-  if (!document.querySelectorAll('.organization').length) {
-    return;
-  }
+  const el = document.querySelector('.page-content.organization.settings');
+  if (!el) return;
 
-  document.querySelector<HTMLInputElement>('.organization.settings.options #org_name')?.addEventListener('input', function () {
-    const nameChanged = this.value.toLowerCase() !== this.getAttribute('data-org-name').toLowerCase();
-    toggleElem('#org-name-change-prompt', nameChanged);
+  queryElems(el, '#org_name', (e: HTMLInputElement) => {
```

Comment:
There won't be more than one elements. So using `queryElems` seems to be an abuse.